.stylelintrc

```diff
@@ -0,0 +1,121 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "customSyntax": "postcss-less",
+  "rules": {
+    "selector-class-pattern": null,
+    "custom-property-pattern": null,
+    "selector-id-pattern": null,
+
+    "rule-empty-line-before": null,
+    "comment-empty-line-before": null,
+    "custom-property-empty-line-before": null,
+    "at-rule-empty-line-before": null,
+    "declaration-empty-line-before": null,
+
+    "property-no-vendor-prefix": null,
+    "alpha-value-notation": null,
+    "color-function-notation": null,
+    "hue-degree-notation": null,
+    "length-zero-no-unit": null,
+
+    "function-no-unknown": [
+      true,
+      {
+        "ignoreFunctions": [
```

Comment:
Grabbed these from https://lesscss.org/functions/ (with one manual correction for `% format` -> `%`):
```js
const functions = Array.from(document.querySelectorAll('.section-content h3.docs-heading'), heading => heading.textContent.replace('\n', ''));
```